plugins/image-prioritizer/tests/test-cases/common-lcp-image-with-fully-incomplete-sample-data.php

```diff
@@ -26,6 +26,7 @@
 			);
 		}
 	},
+	// Note: loading=lazy is not removed from these images URL Metrics are only gathered for desktop so far. Both mobile and desktop URL Metrics are required to proceed with lazy-loading.
```

Comment:
```suggestion
	// Note: loading=lazy is not removed from these images because URL Metrics are only gathered for desktop so far. Both mobile and desktop URL Metrics are required to proceed with lazy-loading.
```